scripts/decode-syseeprom

```diff
@@ -75,7 +80,15 @@ def get_cmdline_opts():
 #
 # Run
 #
-def run(target, opts, args):
+def run(target, opts, args, support_eeprom_db):
+
+    if support_eeprom_db and opts.db:
+        err = target.read_eeprom_db()
+        if err:
+            print "Failed to read EEPROM information from database. Read from cache file"
```

Comment:
This function now returns zero on success but returns nothing on failure. Return value type should be consistent. Either sero/non-zero or true/false. Also, nothing appears to be checking this return value.

Reply:
If you see the latest version of https://github.com/Azure/sonic-platform-common/pull/21, read_eeprom_db() will return -1 for error.